extension.ts

```diff
@@ -407,6 +408,18 @@ export async function activate(context: vscode.ExtensionContext) {
     toggleExtension(configuration.disableExtension, compositionState);
   });
 
+  registerCommand(context, 'vim.vimTutor', async () => {
+    const setting: vscode.Uri = vscode.Uri.parse('untitled:' + 'vimtutor.txt');
+
+    const document = await vscode.workspace.openTextDocument(setting);
+
+    const text = await vscode.window.showTextDocument(document, 1, false);
```

Comment:
Why `ViewColumn = 1` rather than the default (active)? Why `preserveFocus = false`?

Reply:
Ya, I don't need those, just removed it.